src/main/kotlin/org/vividus/testsite/api/HttpRequestController.kt

```diff
@@ -41,6 +41,14 @@ class HttpRequestController {
         return ResponseEntity.ok().build()
     }
 
+    @GetMapping("/redirect")
```

Comment:
can it be parametrised with location? I think it might be useful but it really depends on how you are planning to use it

Reply:
> can it be parametrised with location? I think it might be useful but it really depends on how you are planning to use it

It will be enough for us to receive 302 SC from the server